recipes/r-wicket/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = '0.4.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-wicket
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/wicket_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/wicket/wicket_{{ version }}.tar.gz
+  sha256: ce76a4c8f79e7dfb9d8aadfa59b7f5cdb0de3dd9623b1426fd5caf6464fd3fd2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-bh
+    - r-rcpp
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-bh
```

Comment:
`r-bh` is a build-only dependency, please remove from `run` as it is quite large.

```suggestion
```

Reply:
Nicely spotted, thanks! 🎉